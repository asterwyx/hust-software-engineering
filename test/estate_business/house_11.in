preset user AQ
preset userloc A 10 0
preset map 11 A 0
preset fund A 400
step 1
y
dump
