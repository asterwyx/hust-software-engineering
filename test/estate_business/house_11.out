user AQ
map 11 A 1
userloc A 11 0
userloc Q 0 0
fund A 200
fund Q 10000
credit A 0
credit Q 0
nextuser Q